### PR TITLE
OT-678: Logout

### LINKED ISSUE #294: Logout
**Is your feature request related to a problem? Please describe.**
It's currently not possible to logout of the app.

**Describe the solution you'd like**
A logout should be possible from within the settings.

[Internal bug tracker](https://jira.open-xchange.com/browse/OT-424)

### PR DESCRIPTION
**Pull request by phranck**
### Link to the given issue
Fixes #294.

### Changes
* Adds *Logout* button in user profile
* Adds logout confirmation alert
* Adds native part in Swift (application and plugin) to handle dropping DCC database and termination of the app

### Additional notes
* Changes height of `SettingsItem` and `ListGroupHeader` in consultation with Cornelius
* Adds parameter in `SettingsItem` to enable/disable visibility of right chevron

**NOTE**:
Pull request for [flutter-deltachat-core](https://github.com/open-xchange/flutter-deltachat-core/pull/76) plugin.

### CODE REVIEW

File: lib/src/main/main_bloc.dart [hunk 1]
```diff
@@ -99,7 +110,7 @@ class MainBloc extends Bloc<MainEvent, MainState> {
     if (event is PrepareApp) {
       yield MainStateLoading();
       try {
-        var buildContext = event.context;
```
Comment (florianhaar) [inline lib/src/main/main_bloc.dart]: buildContext was a better name because of clarification.

**Comment by phranck** [comment]
> Looks good. There are many unused imports. Please remove them and look at my other comment :)

Yeah, they were added during lots of tests. Will remove them.

> And please squash your commits and rebase the develop branch.

AFAIK sqashing all commits is being made during merge of this PR into `develop` after aproval, isn't it?

**Comment by phranck** [comment]
FYI: Just ignore the last 3 commits, I made a mistake... 🙄

**Comment by Boehrsi** [comment]
Fixed by https://github.com/open-xchange/ox-coi/commit/f1b40b48de6c82a6236f8764903d69b92ba2af9c

**Pull request closed without merging**